src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -639,7 +659,8 @@ public CustomJenkinsRule() throws Exception {
         }
 
         @Override public URL getURL() throws IOException {
-            return new URL(System.getProperty("RealJenkinsRule.url"));
+            int port = readPort(new File(System.getProperty("winstone.portFileName")));
```

Comment:
Should be unnecessary to use a sysprop.

Reply:
I want something that can be used for ath as well, https://github.com/jenkinsci/acceptance-test-harness/pull/711/

Reply:
doesn't this need to be relative to `JENKINS_HOME`, otherwise you would not be able to start 2 Jenkins with `RealJenkinsRule`?

Reply:
It is initialized at https://github.com/jenkinsci/jenkins-test-harness/pull/348/files#diff-ade203da1dc447eff46cf045f9a21d8ac019ccffdae26359a6838a54a1ac48fbR428, which is within `JENKINS_HOME`